bin/fuel-core/src/cli/run/graphql.rs

```diff
@@ -12,6 +12,10 @@ pub struct GraphQLArgs {
     #[clap(long = "port", default_value = "4000", env)]
     pub port: u16,
 
+    /// The size of the batch fetched from the database by GraphQL service.
+    #[clap(long = "graphql-database-batch-size", default_value = "100", env)]
+    pub database_batch_size: usize,
```

Comment:
does this *have* to be configurable? can we just set it to a global now to reduce the config surface? we have too many options rn :)

Reply:
I decided to make it configurable for now to be able to adjust our performance on the fly. Later we can remove it when we have better ideas about the best value